analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseReturnStatement.vb

```diff
@@ -87,33 +96,50 @@ Public Class DoesNotApplyOn
             .WriteToOtherProperty = 69 ' Compliant
         }
     End Function
-    
+
     Public Function WriteToOtherFunction(other As OtherType) As Integer
         With other
             Return .WriteToOtherFunction(42) ' Compliant
         End With
     End Function
-    
+
     Public Function ImplementedInterfaceMethod() As Integer Implements IInterface.ImplementedInterfaceMethod ' Compliant
         Return 42
     End Function
-    
+
+    <CustomAttribute>   ' Compliant
+    Public Function CustomAttribute() As Integer
+    End Function
+
+    Private Function ArgumentName() As Integer
+        WithExplicitArgumentName(ArgumentName:=42)  ' Compliant
+        WithExplicitArgumentName(ArgumentName:=42)
```

Comment:
line 54 and this are very hard to read because the type of the argument is string... and here you are passing an integer...

why is this compliant?

Reply:
I fixed the `String` issue, that wasn't intended.

It's compliant, because it's parameter name like `param : value` C# syntax. It doesn't use the hidden variable.